pkg/scheduler/cache/node_info.go

```diff
@@ -579,7 +569,6 @@ func (n *NodeInfo) SetNode(node *v1.Node) error {
 		}
 	}
 	n.TransientInfo = newTransientSchedulerInfo()
-	n.updateImageSizes()
```

Comment:
Why can't we set the imageStates here?

Reply:
Good point - so the main reason is that when we add a new node, we need to update and create both the cache.imageStates and the nodeinfo's image states/summaries; and if we set the imageStates in node_info separately, then we need to traverse the new node to find the images twice - one for the cache and the other for nodeinfo.

Thank you for the comments, I will address the above in follow-up PRs.